dev/Effects/microsoft.ui.private.composition.effects_impl.h

```diff
@@ -167,7 +167,7 @@ namespace Microsoft { namespace UI { namespace Private { namespace Composition {
             auto ret = func();
             auto propertyValue = ret.as<winrt::IPropertyValue>();
             to_winrt(*value) = propertyValue;
-            CATCH_RETURN;
```

Comment:
Is CATCH_RETURN defined in WIL?  Curious why we're renaming this.

Reply:
Yes. That is the reason.